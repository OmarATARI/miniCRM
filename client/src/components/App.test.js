import React from 'react';
import ReactDOM from 'react-dom';
import toJson from 'enzyme-to-json';
import App from './App';
import { shallow, mount, render } from 'enzyme';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);   //lÃ¨ve une exception en cas d'erreur
  ReactDOM.unmountComponentAtNode(div);
});


/*Snapshot differences */

// it('renders correctly enzyme', () => {
//   const wrapper = shallow(<App />)
//
//   expect(toJson(wrapper)).toMatchSnapshot();
// });
