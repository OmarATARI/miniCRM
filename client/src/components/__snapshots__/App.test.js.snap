// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly enzyme 1`] = `
<div>
  <div
    className="navbar-fixed"
  >
    <nav
      className="blue lighten-2"
    >
      <div
        className="nav-wrapper"
      >
        <a
          className="brand-logo center"
          href="/"
        >
          Contacts
        </a>
      </div>
    </nav>
  </div>
  <div>
    <Form
      updateData={[Function]}
    />
    <Grid
      deleteData={[Function]}
      items={Array []}
    />
  </div>
</div>
`;
